src/content/docs/developer-tools/kinde-api/access-token-for-api.mdx

```diff
@@ -1,9 +1,10 @@
 ---
-page_id: 50284476-2442-414c-af20-01ed3ef4ca4e
-title: Connect securely to Kinde’s API
+page_id: fa5b704d-1682-47df-ac41-524b56096e79
+title: Make access token request to the Kinde API
```

Comment:
Call Kinde management API with Postman

Reply:
Have renamed it 'Call the Kinde Management API'